src/components/fast_clock.h

```diff
@@ -0,0 +1,133 @@
+/**
+ * @file loconet.h
+ * @brief Loconet base functionality
+ *
+ * \copyright Copyright 2017 /Dev. All rights reserved.
+ * \license This project is released under MIT license.
+ *
+ * This is a basic implementation of the clock system for Loconet.
+ * It reacts on the fast clock messages of loconet to sync the clock.
+ * Internally, it updates the clock, until a new fast clock message
+ * arrives, after which it resets the clock to the received message,
+ * and starts ticking again, using the appropriate clock rate.
+ *
+ * Additionally, the system can run as a clock master, i.e., it sends
+ * the fast clock messages itself. For this, set the appropriate values
+ * using:
+ *
+ *    fast_clock_set_as_master(id1, id2, intermessage_delay)
```

Comment:
If the name of the function is changed, it should be done here as well. Same for slave below.